### PR TITLE
Added additional test case to the Confirm the Ending challenge

### LINKED ISSUE #11341: Code lets pass user even if code is wrong because of inadequate number of  cases
Challenge [Confirm the Ending](https://www.freecodecamp.com/challenges/confirm-the-ending#?solution=%0Afunction%20confirmEnding(str%2C%20target)%20%7B%0A%20%20%2F%2F%20%22Never%20give%20up%20and%20good%20luck%20will%20find%20you.%22%0A%20%20%2F%2F%20--%20Falcor%0A%20%20var%20b%3D3%3B%0A%20%20str%20%3D%20str.split('')%3B%0A%20%20target%20%3D%20target.split('')%3B%0A%20%20if(target.length%20%3D%3D1%20%26%26%20target%5B0%5D%20!%3D%20str%5Bstr.length-1%5D)%0A%20%20%20%20return%20false%3B%0A%20%20%20if(target.length%20%3D%3D1%20%26%26%20target%5B0%5D%20%3D%3D%20str%5Bstr.length-1%5D)%0A%20%20%20%20return%20true%3B%0A%20%20for(i%3D0%3Bi%3Ctarget.length%3Bi%2B%2B)%20%7B%0A%20%20if%20(target%5Btarget.lenghth-i%5D%20%3D%3D%20str%5Bstr.length-i%5D)%0A%20%20%20%20b%2B%2B%3B%0A%20%20%7D%0A%20%20if(i%3D%3Db%20)%0A%20%20%20%20return%20true%3B%0A%20%20else%20%0A%20%20%20%20return%20false%3B%0A%20%0A%7D%0A%0AconfirmEnding(%22dename%22%2C%20%22game%22)%3B%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.143 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```javascript

function confirmEnding(str, target) {
  // "Never give up and good luck will find you."
  // -- Falcor
  var b=3;
  str = str.split('');
  target = target.split('');
  if(target.length ==1 && target[0] != str[str.length-1])
    return false;
   if(target.length ==1 && target[0] == str[str.length-1])
    return true;
  for(i=0;i<target.length;i++) {
  if (target[target.lenghth-i] == str[str.length-i])
    b++;
  }
  if(i==b )
    return true;
  else 
    return false;
 
}

confirmEnding("dename", "game");

```

### PR DESCRIPTION
**Pull request by ghost**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #11341

### CODE REVIEW

**Comment by Bouncey** [comment]
@PedroCSSilva sorry to be a pain, but can you please add the test suggested by @systimotic [here](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/11341#issuecomment-255689651).

Many thanks :+1:

File: seed/challenges/01-front-end-development-certification/basic-bonfires.json [hunk 1]
```diff
@@ -374,7 +374,9 @@
         "assert(confirmEnding(\"He has to give me a new name\", \"name\") === true, 'message: <code>confirmEnding(\"He has to give me a new name\", \"name\")</code> should return true.');",
         "assert(confirmEnding(\"Open sesame\", \"same\") === true, 'message: <code>confirmEnding(\"Open sesame\", \"same\")</code> should return true.');",
         "assert(confirmEnding(\"Open sesame\", \"pen\") === false, 'message: <code>confirmEnding(\"Open sesame\", \"pen\")</code> should return false.');",
+        "assert(confirmEnding(\"Open sesame\", \"game\") === false, 'message: <code>confirmEnding(\"Open sesame\", \"same\")</code> should return true.');",
```
Comment (Bouncey) [inline seed/challenges/01-front-end-development-certification/basic-bonfires.json]: Thank you for updating 👍 

Can you amend the message to `should return false` please?

We will be good for a merge after this change :tada:

Comment (systimotic) [inline seed/challenges/01-front-end-development-certification/basic-bonfires.json]: I think you just forgot to update the message 😄 When you change it to return false, can you also change the `same` to `game` in the message like it is in the test? Thank you for your contribution!

Comment (PedroCSSilva) [inline seed/challenges/01-front-end-development-certification/basic-bonfires.json]: Sorry guys for the error. I had a 2min break during work and decided to update the PR.

I'll make sure everything is ok soon.

**Comment by systimotic** [comment]
@PedroCSSilva 👍 We're getting really close! The message looks good to me. There's just one thing left before it's ready to be merged, [squashing your commits](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231). Can you please check out that link and update the Pull Request? Thanks in advance!

**Comment by PedroCSSilva** [comment]
I hope everything is ok now. :)

This was an awesome experience guys, thanks for everything. Learned something new about git (squash)

**Comment by systimotic** [comment]
It looks good to me! @Bouncey I think this is ready to be merged. Thank you for contributing Pedro!

**Comment by Bouncey** [comment]
## LGTM :+1: 

Thanks for contributing :tada:

**Pull request merged**